extension/src/plots/model/index.ts

```diff
@@ -154,44 +153,40 @@ export class PlotsModel extends ModelWithPersistence {
   }
 
   public getCustomPlots(): CustomPlotsData | undefined {
-    const experiments = this.experiments
-      .getWorkspaceCommitsAndExperiments()
-      .filter(({ id }) => id !== EXPERIMENT_WORKSPACE_ID)
-
-    if (experiments.length === 0) {
-      return
-    }
-
+    const experiments = this.experiments.getUnfilteredCommitsAndExperiments()
+    const hasUnfilteredExperiments = experiments.length > 0
+    const plotsOrderValues = this.getCustomPlotsOrder()
+    const enablePlotCreation = checkForCustomPlotOptions(
+      this.experiments.getColumnTerminalNodes(),
+      plotsOrderValues
+    )
     const height = this.getHeight(PlotsSection.CUSTOM_PLOTS)
     const nbItemsPerRow = this.getNbItemsPerRowOrWidth(
       PlotsSection.CUSTOM_PLOTS
     )
+    const hasAddedPlots = plotsOrderValues.length > 0
+
     const colorScale = getColorScale(
       this.getSelectedRevisionDetails().filter(
         ({ id }) => id !== EXPERIMENT_WORKSPACE_ID
       )
     )
-    const plotsOrderValues = this.getCustomPlotsOrder()
-    const plots: CustomPlotData[] = collectCustomPlots({
-      colorScale,
-      experiments,
-      height,
-      nbItemsPerRow,
-      plotsOrderValues
-    })
-
-    if (plots.length === 0 && plotsOrderValues.length > 0) {
```

Comment:
It's actually not possible for this if statement to be true. `collectCustomPlots` creates an array that matches the order of `plotsOrderValues`. Aka they will always be the same length. I went ahead and removed it.